Salesengine iteration 0

notmarkmiranda opened the pull request:
THIS IS NOT COMPLETELY FINISHED, DO NOT MERGE!

@jeneve, I just wanted to put this code up. I got it working. I had to iterate through each side of the csv load.There may be room for refactoring, but I just need to make sure I can test all the functions out. I'll explain what i did in the morning.

jeneve commented:
Cool! Yours is totally different from mine, but it works, so it's better! We'll refactoring and merge it in the am when I am rested enough to understand it.

notmarkmiranda commented:
This is ready to go!
- Added the 'coverage' folder to our '.gitignore' file to prevent conflicts with testing updating coverage
- Updated rakefile to be able to run all tests by executing `rake test` from the command line

notmarkmiranda commented:
added comment for later refactoring in sales engine
